### PR TITLE
Section claudia

### PR DESCRIPTION
**Pull request by Clusi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,2 @@
-# Conference Management System
```
Comment (raress96) [inline README.md]: Nu stiu de ce ai facut asta dar da undo :P

File: src/main/java/domain/Section.java [hunk 2]
```diff
@@ -1,24 +1,119 @@
...
+
+    @ManyToOne
+    @JoinColumn(name = "conference_id",
+            foreignKey = @ForeignKey(name = "CONFERENCE_ID_FK")
+    )
+    private Conference conference;
+
+    public Section() {this(null,null);}
+
+    public Section( String name, String room) {
```
Comment (raress96) [inline src/main/java/domain/Section.java]: Userul si conferinta de ce nu ii aici?Si nu lasa asa spatii, toti ar trebui sa folosim aceleasi standarde de cod.

**Review by raress96 — CHANGES_REQUESTED** [review]

**Comment by Clusi** [comment]
pai userul si conferinta nu-s acolo? eu nu inteleg ce nu e acolo?

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2017-04-04 16:50 GMT+03:00 Rares Serban <notifications@github.com>:

File: src/main/java/domain/Section.java (hunk 2 above)
Comment (raress96) [reply src/main/java/domain/Section.java]: Nu sunt ca parametrii la constructor si vezi ca aici trebuia lasat comentariu

**Pull request merged**